src/main/java/Ixo.java

```diff
@@ -0,0 +1,151 @@
+import java.util.Scanner;
+
+
+public class Ixo {
+
+    public final static String SEPARATOR = "____________________________________________________";
+
+    public static void menu() {
+        System.out.println("1. Task List");
+    }
+
+    public static void taskList() {
+
+        Scanner lineScan;
+        Task[] taskStore = new Task[100]; //limit list to store only 100 items
+        int storeIndex = 0;
+        int taskToMark;
+        String cmd;
+        String inputLine;
+        String[] content;
+
+
+        System.out.println("I'll now take a note on what you typed. Type 'bye' by itself to make me stop.");
+
+        while (true) {
+            System.out.println(SEPARATOR);
+            lineScan = new Scanner(System.in);
+            System.out.print("");
+            inputLine = lineScan.nextLine();
+            cmd = inputLine.split(" ")[0];
+
+            switch (cmd.toLowerCase()) {
+            case "bye":
+                return;
+            case "list":
+                System.out.println("Here's your list of tasks: ");
+                int i = 1;
+                for (Task item : taskStore) {
+                    if (item == null) {
+                        break;
+                    }
+                    System.out.println(i + ": " + item);
+                    i++;
+                }
+                break;
+
+            case "mark":
+            case "unmark":
+                try {
+                    taskToMark = Integer.parseInt(inputLine.split(" ")[1]) - 1; //-1 to match up user view to actual index access
+                    if (taskToMark >= storeIndex || taskToMark < 0) {
+                        System.out.println("that task does not exist");
+                        System.out.println(SEPARATOR);
+                    }
+                    boolean isMark = cmd.equals("mark");
+                    if (isMark) {
+                        System.out.println("Well done! task " + taskStore[taskToMark] + " has been marked.");
+                    } else {
+                        System.out.println("The task " + taskStore[taskToMark] + " has been unmarked.");
+                    }
+                    taskStore[taskToMark].isDone = isMark;
+                } catch (NumberFormatException e) { //for non-number input
+                    System.out.println("You did not enter a number");
+                } catch (ArrayIndexOutOfBoundsException e) { //for out of bounds index entered
+                    System.out.println("You did not enter a valid corresponding number");
+                }
```

Comment:
This is deep nesting and can borderlines arrowhead code. Refactor the internal indents to another function.